modules/nf-core/parabricks/fq2bam/tests/main.nf.test

```diff
@@ -0,0 +1,93 @@
+nextflow_process {
+
+    name "Test Process PARABRICKS_FQ2BAM"
+    script "../main.nf"
+    process "PARABRICKS_FQ2BAM"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "parabricks"
+    tag "parabricks/fq2bam"
+
+    test("parabricks_fq2bam") {
+
+
+    test("parabricks_fq2bam_pe_default") {
```

Comment:
there is something off?

Reply:
indeed, we had typing errors that have been resolved now